requirements/edx/development.txt

```diff
@@ -351,3 +351,5 @@ xblock==1.2.1
 xmltodict==0.11.0
 zendesk==1.1.1
 zope.interface==4.5.0
+django-tinymce==2.8.0
```

Comment:
Based on `make upgrade` rule new requirements must be placed in `development.in` and then `development.txt` file must auto-generated.
`*.txt` files can't be edited manually.